templates/user/auth/captcha.tmpl

```diff
@@ -0,0 +1,27 @@
+{{if and .EnableCaptcha (eq .CaptchaType "image")}}
```

Comment:
I know that's just the old code but could just put a single `{{if .EnableCaptcha}}` on top. Is easier to read imho.

Reply:
I've done this for you @xin-u